remoulade/middleware/retries.py

```diff
@@ -84,7 +89,16 @@ def after_process_message(self, broker, message, *, result=None, exception=None)
         message.options["traceback"] = traceback.format_exc(limit=30)
         min_backoff = self.get_option("min_backoff", broker=broker, message=message)
```

Comment:
it can't work as `DEFAULT_MIN_BACKOFF` is used if None, it should be managed as default here no?